test/README.md

```diff
@@ -26,12 +26,43 @@ Make sure you have valid credentials for accessing GCP:
 gcloud auth login
 ```
 
+Set the project property in the core section by running:
+```
+gcloud config set project PROJECT_ID
+```
+
+Install jq by running:
+```
+sudo apt-get install jq
+```
+
 Make sure you are at the root directory of the repository.
 
 ```
 cd gke-networking-recipes
 ```
 
+
+Make sure to set ZONE and REGION environment variables before running tests. Resources will be deployed to the specified region and/or zone.
+```
+export ZONE=zone
+export REGION=region
+```
+
```

Comment:
This is hard to maintain in a single file. Move the content specific to the test to a README in the test's directory.